packages/blockchain-wallet-v4-frontend/src/data/modules/profile/sagas.ts

```diff
@@ -730,10 +729,15 @@ export default ({ api, coreSagas, networks }) => {
       )
 
       let sofiUserState = response.sofiJwtPayload?.state
+      let sofiUserCountry = response.sofiJwtPayload?.country
```

Comment:
wouldn't this work too, and cleaner?

`let sofiUserCountry = response.sofiJwtPayload?.country || 'US'`

Reply:
Done